tools/rack/rack.plugin/build.properties

```diff
@@ -53,7 +53,11 @@ bin.includes = .,\
                plugin.xml,\
                splash.bmp,\
                RITE-16.bmp,\
-               RITE-32.bmp
+               RITE-32.bmp,\
+               lib/javafx-swing.jar,\
+               lib/javafx-swing-linux.jar,\
+               lib/javafx-swing-mac.jar,\
+               lib/javafx-swing-win.jar
```

Comment:
Likewise, please keep these filenames sorted alphabetically so maintainers can check for missing files more easily.

Reply:
Fixed.